content/applications/websites/livechat/responses.rst

```diff
@@ -1,48 +1,177 @@
+view of the chat.
+
+.. image:: responses/responses-ticket-link.png
+   :align: center
+   :alt: View of the chat window with a helpdesk ticket created in Odoo Live Chat.
+
+More information about each available command can be found below.
+
+Help
+----
+
+If an operator types `/help` in the chat window, an informative message that includes the potential
+entry types an operator can make is displayed.
+
+.. image:: responses/responses-help.png
+   :align: center
+   :alt: View of the message generated from using the /help command in Odoo Live Chat.
+
+.. seealso::
+  - :doc:`/applications/productivity/discuss/overview/get_started`
+  - :doc:`/applications/productivity/discuss/overview/team_communication`
+
+Helpdesk & Helpdesk search
+--------------------------
+
+The `/helpdesk` and `/helpdesk_search` commands allow operators to both create helpdesk tickets
+directly from a conversation, and search through existing tickets by keyword or ticket number.
 
 .. important::
-   | - For *helpdesk tickets*: make sure the application is installed on your database and the
-     option *Live Chat* under :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`
-     is enabled.
-   | - For *leads*: the *CRM* application must be installed on your database.
+   The `/helpdesk` and `/helpdesk_search` commands can only be used if the *Helpdesk* app has been
+   installed, and *Live Chat* has been activated on a *Helpdesk Team*. To activate :guilabel:`Live
+   Chat`, go to :menuselection:`Helpdesk application --> Configuration --> Teams`, and select a
+   team. Scroll to the :guilabel:`Channels` section and check the box labeled :guilabel:`Live Chat`.
 
-To access the ticket or lead created from the chat, click on the shortcut link.
+Create a ticket from a live chat
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. image:: responses/create_ticket.png
+If an operator types `/helpdesk` in the chat window, the conversation is used to create a *Helpdesk
+Ticket*.
+
+After entering the `/helpdesk` command, select a title for the ticket and type it into the chat
+window, then press `Enter`.
+
+.. image:: responses/helpdesk.png
    :align: center
-   :alt: View of the chat window with a helpdesk ticket created in Odoo Live Chat
+   :alt: View of the results from a helpdesk search in a Live Chat conversation.
 
-.. tip::
-   Helpdesk tickets created from the chat automatically add the conversation as a description of
-   the ticket. The same goes for the creation of a lead.
+The newly created ticket will be added to the *Helpdesk* team that has live chat enabled. If more
+than one team has live chat enabled, the ticket will automatically be assigned based on the team's
+priority.
+
+The transcript from the conversation will be added to the new ticket, under the
+:guilabel:`Description` tab.
+
+To access the new ticket, click on the link in the chat window, or go to the
+:menuselection:`Helpdesk app` and click the :guilabel:`Tickets` button on the kanban card for the
```

Comment:
Keep "app" out of the menuselection where applicable 
```suggestion
To access the new ticket, click on the link in the chat window, or go to the
:menuselection:`Helpdesk` app and click the :guilabel:`Tickets` button on the kanban card for the
```

Reply:
This is up to the contributor actually; we don't have a hard guideline on this.

Reply:
Ah ok understood, sorry @jero-odoo! So to be clear, the following are ok:
```
go to the :menuselection:`Helpdesk app` and click the ✅
```
and
```
go to the :menuselection:`Helpdesk` app and click the ✅
```
As well as with italics?
```
This can also be done from the *Helpdesk app* by navigating to... ❓
```